src/ast/dml.rs

```diff
@@ -236,19 +239,6 @@ impl Display for CreateTable {
             HiveDistributionStyle::PARTITIONED { columns } => {
                 write!(f, " PARTITIONED BY ({})", display_comma_separated(columns))?;
             }
-            HiveDistributionStyle::CLUSTERED {
```

Comment:
`PARTIONED BY` and `CLUSTERED BY`can appear at the same time, so we cannot use an enum to put them together. I remove `CLUSTERED` from HiveDistributionStyle here but it won't affect users since we don't support parsing `CLUSTERED BY` before this PR. cc @iffyio